Metrics.md

```diff
@@ -1,5 +1,166 @@
 For access to the `/api/v2/metrics` endpoints you must be [Authenticated](Authentication.md)
 
+#### GET `/api/v2/metrics/stats`
+
+The main metrics endpoint
+
+Query Params:
+
+|Query Param|Example|Notes|
+|:---|:---|:---|
+|metric|`scans`|`String` - Required. Valid metrics: `scans`, `taps`|
+|from*|`1521561000000`|`Number` - Required. Date in Milliseconds|
+|to*|`1524153000000`|`Number` - Required. Date in Milliseconds. to _must_ be after from|
+|dimension|`activity`|`String` - Required. Valid dimensions: `activity`, `phyid`, `app`, `destination`, `os`|
+|value|`["abc123"]`|`String`or `Array` - Required. Valid values are `all` or an array of IDs for the given dimension|
+|interval|`hour`|`String` - Required. Valid intervals: `15min`, `hour`, `day`, `week`, `month`, `quarter`, `year`|
+|touchpoint|`abc123`|`String` - Optional, will be deprecated in favor of more robust dimensions. Secondary dimension to pin down further on `destination` dimension|
+
+*NOTE: `stats` is limited by the number of responses based on the `interval`, `from`, and `to`, with the following table:
+ 
+|Interval| Max Range|
+|:---:|:---:|
+|`15min`|24 hours|
+|`hour`|3 days|
+|`day`|91 days|
+|`week`|1 year|
+|`month`|2 years|
+|`quarter`|4 years|
+
+Example Request: 
+```
+https://my.phy.net/api/v2/metrics/stats?metric=scans&from=1521561000000&to=1524153000000&dimension=activity&value=all&interval=day
```

Comment:
Are 'from' and 'to' supposed to be swapped here?

Reply:
The request I was using that is working is structured a little different 
"https://my.phy.net/api/v2/metrics/stats?metric=scans&dimension=phyid&value=all&to=1523982165633&from=1523896360822&interval=day"